src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -114,7 +114,7 @@ class Docker implements Serializable {
         
         public <V> V inside(String args = '', Closure<V> body) {
             docker.node {
-                if (docker.script.sh(script: "docker inspect -f . ${id} >&- 2>&-", returnStatus: true) != 0) {
+                if (docker.script.sh(script: "docker inspect -f . ${id}", returnStatus: true) != 0) {
```

Comment:
This is needed by this PR iirc, it was causing the build to fail when using the demo container.